stable/parse/templates/ingress.yaml

```diff
@@ -17,7 +17,8 @@ metadata:
     {{- end }}
 spec:
   rules:
-  {{- range .Values.ingress.dashbard.hosts }}
+  {{- if .Values.dashboard.enabled }}
```

Comment:
Shouldn't you include this check at the top of file?

Reply:
`ingress.yaml` actually contains two rules, one for the dashboard and one for the server.

I think that the rule for dashboard should be excluded from the spec if it's not enabled. However the ingress for parse-server should still be generated if `.Values.ingress.enabled` eq `true`.  
Do you not agree or should this be two separate ingress templates?

I also notice that i've missed a part that's still being generated and will fix this asap.

Reply:
Oh! You're right! I forgot the same Ingress is used for both Parse and Parse Dashboard. In that case, it's fine with the approach you used.